heartbeat/galera

```diff
@@ -880,8 +1169,18 @@ fi
 case "$1" in
   start)    galera_start;;
   stop)     galera_stop;;
-  status)   mysql_common_status err;;
-  monitor)  galera_monitor;;
+  status)   galera_status err;;
+  monitor)
```

Comment:
I'm not sure how to convince pacemaker to call the resource agent with a specific action name when a probe is required, or when the resource is in a certain state (e.g. "resource is currently unmanaged on that node")
Anyway, seems that pcs is a limiting layer here :/